src/Compute/v2/Api.php

```diff
@@ -668,4 +668,66 @@ public function getHypervisor(): array
             'params' => ['id' => $this->params->urlId('hypervisor')]
         ];
     }
+
+    public function getQuotaSet()
+    {
+        return [
+            'method' => 'GET',
+            'path'   => 'os-quota-sets/{tenantId}',
+            'params' => [
+                'tenantId' => $this->params->urlId('quota-sets')
+            ]
+        ];
+    }
+
+    public function getQuotaSetDetail()
+    {
+        return [
+            'method'  => 'GET',
+            'path'    => 'os-quota-sets/{tenantId}/detail',
```

Comment:
this is fine, but if you have large operation data structs in the future, it's better to do:

```php
public function getQuotaSetDetail()
{
   $data = self::getQuotaSet();
   $data['url'] += '/detail';
   return $data;
}
```